app/renderer/js/pages/preference/find-accounts.ts

```diff
@@ -24,15 +29,15 @@ class FindAccounts extends BaseComponent {
 		`;
 	}
 
-	init() {
+	init(): void {
 		this.$findAccounts = this.generateNodeFromTemplate(this.template());
-		this.props.$root.appendChild(this.$findAccounts);
+		this.props.$root.append(this.$findAccounts);
 		this.$findAccountsButton = this.$findAccounts.querySelector('#find-accounts-button');
-		this.$serverUrlField = this.$findAccounts.querySelectorAll('input.setting-input-value')[0];
+		this.$serverUrlField = this.$findAccounts.querySelectorAll('input.setting-input-value')[0] as HTMLInputElement;
```

Comment:
Do we really need to do `as HTMLInputElement` here? I feel like this should be unnessacary since we declared above that this variable is in fact type of `HTMLInputElement`.

Reply:
Then, we get this-
```
Type 'Element' is missing the following properties from type 'HTMLInputElement': accept, align, alt, autocomplete, and 164 more.ts(2740)
```

Reply:
OK, so we do need this.